build.gradle

```diff
@@ -10,7 +19,7 @@ subprojects {
     apply plugin: 'java-library-distribution'
     apply plugin: 'com.diffplug.spotless'
 
-    project.version = '4.0.0-SNAPSHOT'
+    project.version = projectVersion
```

Comment:
Can we also replace the following with `projectGroup`?
https://github.com/scalar-labs/scalardl/blob/5811a2cea8e197890bf6bd0f31544449377b6001/build.gradle#L97

Reply:
Good catch! Fixed in 731993f. Thank you!